### ISSUE TITLE
Rename tooltips are persistent

### ISSUE THREAD

**Issue by reinaH** [CenterForOpenScience/osf.io#2844]
Reported on the 0.37 Trello board

This problem appears in Firefox and Chrome

### Rename tooltips are persistent in Chrome and Firefox

# Steps
1. Go to a project on which you have admin or read/write access
2. In either the files tab or the files grid on the project overview page, click to select a file/folder
3. In the toolbar, look for the Rename option
4. Hover over the option to see the black tooltip (may not show upon first file/folder selected, but this is a separate issue: #xyz. If it does not show for first file/folder, click another)
5. Click to rename a file/folder 
6. Once renaming is complete, click a row for which Renaming is not an option (for example, click a row with an add-on name and icon)

# Expected
I expect that once renaming is complete, the tooltip will disappear

# Actual
The tooltip is persistent and does not disappear until refresh. 

#### When hovering over Rename button: 
![gg](https://trello-attachments.s3.amazonaws.com/5564d88252fed1ff1794be53/583x191/a4e05b188e0e0345493113513832518b/Screen_Shot_2015-05-26_at_4.48.33_PM.png)

#### After renaming and selecting another row: 
![ua](https://trello-attachments.s3.amazonaws.com/5564d88252fed1ff1794be53/570x157/ec2ecbb890f1803e126a2c7467468113/Screen_Shot_2015-05-26_at_4.49.15_PM.png)

**Comment by lbanner** [comment]
@caneruguz -- can you check to see if this issue "persists" -- Thanks.

**Comment by caneruguz** [comment]
@lbanner it's not fixed there yet because the PR that I have for removing the tooltip is not merged in yet. 
https://github.com/CenterForOpenScience/osf.io/pull/2842

**Comment by lbanner** [comment]
:ok_hand:

**Comment by caneruguz** [comment]
@lbanner the PR is merged, ready to test again.

**Comment by lbanner** [comment]
On Staging?

**Comment by caneruguz** [comment]
Hmm not sure, @sloria would know.

**Comment by chennan47** [comment]
It is merged into develop, so maybe on staging if staging is branched on develop.

**Comment by lbanner** [comment]
@icereval ? Merge status above ^^ ?

**Comment by sloria** [comment]
PR is merged, but may still need to be put on staging.
